quackfactory/cogs/session.py

```diff
@@ -0,0 +1,34 @@
+from discord.ext import commands
+import discord
+import logging
+
+intents = discord.Intents.default()
+intents.reactions = True
+intents.members = True
+client = discord.Client(intents=intents)
+
+class Session(commands.Cog):
+    def __init__(self, bot):
+        self.bot = bot
+
+@client.event
+async def on_voice_state_update(member, before, after):

```

Comment:
Due to the nature of Python relying on indentation to recognise which methods belong to which class, you need to indent this to be inside of the session class.